Tasks/AzureCLI/azureclitask.ts

```diff
@@ -15,7 +15,12 @@ export class azureclitask {
         try {
             tl.setResourcePath(path.join( __dirname, "task.json"));
 
-            var bash = tl.createToolRunner(tl.which("bash", true));
+            var tool;
+            var scriptType: string = tl.getInput("scriptType", true);
```

Comment:
scriptType removed from the task. Now judging on the basis of OS.
@bryanmacfarlane